autoscale/src/main/java/com/sequenceiq/periscope/monitor/AbstractMonitor.java

```diff
@@ -64,7 +64,7 @@ protected EvaluatorExecutor getEvaluatorExecutorBean(M monitored) {
 
     protected abstract List<M> getMonitored();
 
-    protected abstract M save(M monitored);
+    protected abstract void save(M monitored);
 
```

Comment:
The returned object is unused and avoid additional DB Hit in ClusterMonitor.

Reply:
Will this change affect HA deployments, and failover?